packages/ember/addon/instance-initializers/sentry-performance.ts

```diff
@@ -355,6 +354,26 @@ function _instrumentInitialLoad(config: EmberSentryConfig) {
   performance.clearMeasures(measureName);
 }
 
+function _hasPerformanceSupport() {
```

Comment:
@AbhiPrasad I ended up extracting this out like this. How do you normally handle it when checking something where TS "thinks" it cannot be undefined, but it can on some browsers? It didn't feel quite right to just cast `const performance = window.performance as { ... }`, as then you also need to do e.g. `performance.clearMarks!()` everywhere etc, which is also a bit weird 😅

Reply:
We don’t do this very well, I would just best effort it tbh.